words/words.md

```diff
@@ -150,36 +150,36 @@ die <b style='color:#888'>Graue Kurve</b> gibt die *gesamte* Fälle (aktuell + g
 		<iframe src="sim?stage=epi-5" width="800" height="540"></iframe>
 </div>
 
-Es ändert sich nicht viel! Wie lange Sie <icon e></icon> exponiert bleiben, ändert zwar das Verhältnis von <icon e></icon> zu <icon i></icon> und *wann* die aktuellen Fälle ihren Höhepunkt erreichen... aber die *Höhe* dieser Spitze und die Gesamtzahl der Fälle bleibt am Ende gleich.
+Es ändert sich nicht viel! Wie lange ein <icon e></icon> exponiert bleibt, ändert zwar das Verhältnis von <icon e></icon> zu <icon i></icon> und *wann* die aktuellen Fälle ihren Höhepunkt erreichen... aber die *Höhe* dieser Spitze und die Gesamtzahl der Fälle bleibt am Ende gleich.
 
 Warum das? Wegen der *wichtigsten* Idee im 1x1 der Epidemiologie:
 
 ![](pics/r.png)
 
-Abkürzend für Reproduktionszahl. Sie beschreibt die *durchschnittliche* Anzahl an Menschen, die ein <icon i></icon> infiziert *bevor* dieser genest (oder stirbt).
+Kurz für Reproduktionszahl. Sie beschreibt die *durchschnittliche* Anzahl an Menschen, die ein <icon i></icon> infiziert *bevor* er sich erholt (oder stirbt).
 
 ![](pics/r2.png)
 
 **R** verändert sich im Laufe des Ausbruchsgeschehens durch gesteigerte Immunität und Interventionen. 
 
-**R<sub>0</sub>** (Aussprache R-Null) ist die Reproduktionszahl zum *Beginn eines Ausbruchs, also vor einer Immunität oder von Intenventionen* R<sub>0</sub> spiegelt damit die Stärke des Virus wider, jedoch schwankt diese Zahl von Ort zu Ort. Beispielsweise ist sie in dicht besiedelten Städten höher als in ländlichen Gegenden. 
+**R<sub>0</sub>** (ausgesprochen R-Null) ist die Reproduktionszahl zum *Beginn eines Ausbruchs, also vor einer Immunität und vor Intenventionen* R<sub>0</sub> spiegelt damit die Stärke des Virus wider, jedoch schwankt diese Zahl von Ort zu Ort. Beispielsweise ist sie in dicht besiedelten Städten höher als in ländlichen Gegenden. 
```

Comment:
Korrekt wäre hier der Begriff der Basisreproduktionszahl, oder?

Reply:
Bin in der Terminologie nicht so fest, klingt mir aber schlüssig, ja.

Reply:
Ja, dass ist der richtige Begriff für R_0, aber der wird hier ja auch mit "Reproduktionszahl zum Beginn eines Ausbruchs" korrekt umschrieben, würde also nichts ändern.